tests_new/integration_tests/modules/notifications/queries.py

```diff
@@ -0,0 +1,98 @@
+# TODO: This file will be replaced by using the SDK directly
+
+
+def mark_notification_read(client, uri):
+    query = {
+        'operationName': 'markNotificationAsRead',
+        'variables': {'notificationUri': uri},
+        'query': """
+            mutation markNotificationAsRead($notificationUri: String!) {
+              markNotificationAsRead(notificationUri: $notificationUri)
+            }
+                """,
+    }
+    response = client.query(query=query)
+    return response.data.markNotificationAsRead
+
+
+def delete_notification(client, uri):
+    query = {
+        'operationName': 'deleteNotification',
+        'variables': {'notificationUri': uri},
+        'query': """
+            mutation deleteNotification($notificationUri: String!) {
+              deleteNotification(notificationUri: $notificationUri)
+            }
+                """,
+    }
+    response = client.query(query=query)
+    return response.data.deleteNotification
+
+
+def list_notifications(client, filter={}):
+    query = {
+        'operationName': 'listNotifications',
+        'variables': {'filter': filter},
+        'query': """
+            query listNotifications($filter: NotificationFilter) {
+              listNotifications(filter: $filter) {
+                count
+                page
+                pages
+                hasNext
+                hasPrevious
+                nodes {
+                  notificationUri
+                  message
+                  type
+                  is_read
+                  target_uri
+                }
+              }
+            }
+                """,
+    }
+    response = client.query(query=query)
+    return response.data.listNotifications
+
+
+def count_unread_notificiations(client):
+    query = {
+        'operationName': 'countUnreadNotifications',
+        'variables': {},
+        'query': """
+            query countUnreadNotifications {
+              countUnreadNotifications
+            }
+                """,
+    }
+    response = client.query(query=query)
+    return response.data.countUnreadNotifications
+
+
```

Comment:
As commented already, `count_read_notificiations` and `count_deleted_notificiations` are not used in the FE

Reply:
removed all 3 ops